cli/command/orchestrator/orchestrator.go

```diff
@@ -0,0 +1,53 @@
+package orchestrator
```

Comment:
Could this be in `cli/command` package, or are there issues with circular imports?